### DISCUSSION TITLE
How can I make the text font size  more larger?

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#4226] [Q&A] [answered]
# How to make font size 72px

`
verTit := canvas.NewText("v0.0.1", color.NRGBA{R: 0, G: 0, B: 0, A: 255})
`

# How to do it, is there any reference? Thanks

`
widget.NewLabelWithStyle("One", fyne.TextAlignLeading, fyne.TextStyle{Bold: false})
`

**Comment by andydotxyz** [comment, marked as answer]
The `canvas.Text` and `widget.Label` are different. Canvas package is configurable primitives and the widget package always follows the theme.

For text just set `Text.TextSize` and for `Label` either set the font of your theme to be larger or use `RichText` where you could have a heading segment, for example.

**Reply by junmaqiang** [reply]
thank you answer me  `verTit.TextSize = 37`
